Aug 04, 2025 12:07:51 PM liquibase.changelog
INFO: Reading resource: db/changelog/migrations/db.changelog-2508021213.sql
Aug 04, 2025 12:07:51 PM liquibase.changelog
INFO: Reading resource: db/changelog/migrations/db.changelog-2508021605.sql
Aug 04, 2025 12:07:51 PM liquibase.changelog
INFO: Reading resource: db/changelog/migrations/db.changelog-2508021612.sql
Aug 04, 2025 12:07:51 PM liquibase.changelog
INFO: Reading resource: db/changelog/migrations/db.changelog-2508021615.sql
Aug 04, 2025 12:08:01 PM liquibase.changelog
INFO: Reading from meubanco.DATABASECHANGELOG
Database is up to date, no changesets to execute
Aug 04, 2025 12:08:01 PM liquibase.changelog
INFO: Reading from meubanco.DATABASECHANGELOG

UPDATE SUMMARY
Run:                          0
Previously run:               4
Filtered out:                 0
-------------------------------
Total change sets:            4

Aug 04, 2025 12:08:01 PM liquibase.util
INFO: UPDATE SUMMARY
Aug 04, 2025 12:08:01 PM liquibase.util
INFO: Run:                          0
Aug 04, 2025 12:08:01 PM liquibase.util
INFO: Previously run:               4
Aug 04, 2025 12:08:01 PM liquibase.util
INFO: Filtered out:                 0
Aug 04, 2025 12:08:01 PM liquibase.util
INFO: -------------------------------
Aug 04, 2025 12:08:01 PM liquibase.util
INFO: Total change sets:            4
Aug 04, 2025 12:08:01 PM liquibase.util
INFO: Update summary generated
Aug 04, 2025 12:08:01 PM liquibase.lockservice
INFO: Successfully released change log lock
Aug 04, 2025 12:08:01 PM liquibase.command
INFO: Command execution complete
java.sql.SQLNonTransientConnectionException: No operations allowed after connection closed.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:111)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:98)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:90)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:64)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:74)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.commit(ConnectionImpl.java:803)
	at board.dio.persistence.migration.MigrationStrategy.executeMigration(MigrationStrategy.java:40)
	at board.dio.App.main(App.java:17)
Caused by: com.mysql.cj.exceptions.ConnectionIsClosedException: No operations allowed after connection closed.
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:62)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:150)
	at com.mysql.cj.NativeSession.checkClosed(NativeSession.java:753)
	at com.mysql.cj.jdbc.ConnectionImpl.checkClosed(ConnectionImpl.java:565)
	at com.mysql.cj.jdbc.ConnectionImpl.commit(ConnectionImpl.java:760)
	... 2 more
